src/data/constants.js

```diff
@@ -51,3 +51,69 @@ export const RESERVED_QUICKSTART_IDS = {
   GUIDED_INSTALL: 'GUIDED_INSTALL',
   BUILD_YOUR_OWN_QUICKSTART: 'BUILD_YOUR_OWN_QUICKSTART',
 };
+
+export const QUICKSTARTS_QUERY = `
```

Comment:
Let's move this over to the `instant-observability` page so that changes to this query are picked up by the development server.

Reply:
done!